docs_src/OVMS/pipelinesetup.md

```diff
@@ -1,96 +1,96 @@
 
-```bash
-docker build -f Dockerfile.bitModel -t bit_model_downloader:dev .
-```
+    !!! Note 
+        Optional to install Golang 1.20
 
-Here is the script to run container and downloads the bit models:
+    !!! Debug
+        For the development debugging purpose like running and debugging in IDE, you may want to install Golang 1.20. Here is [the link to download](https://go.dev/dl/) Golang. Follow the [installation instruction](https://go.dev/doc/install#Go_installation) with the same downloaded file name above for version 1.20.
 
-```bash
-docker run -it bit_model_downloader:dev
-```
+3. Build the benchmark Docker images
 
-### Step 7: Build the reference design images
+    ```bash
+    cd benchmark-scripts
+    make build-all
+    ```
 
-You must build the provided component services and create local docker images. Below is the table for the OVMS Server and Client build command:
+4. Download the models manually (Optional)
 
-| Target                            | Docker Build Command               | Check Success                                                          |
-| ----------------------------------| -----------------------------------|------------------------------------------------------------------------|
-| OVMS Server                       | <pre>make build-ovms-server</pre>  | docker images command to show <b>openvino/model_server-gpu:latest</b>  |
-|                                   |                                    | docker images command to show <b>openvino/model_server:latest-gpu</b>  |
-|                                   |                                    | docker images command to show <b>openvino/model_server:latest</b>      |
-|                                   |                                    | docker images command to show <b>openvino/model_server-pkg:latest</b>  |
-|                                   |                                    | docker images command to show <b>openvino/model_server-build:latest</b>|
-| OVMS Profile Launcher             | <pre>make build-profile-launcher</pre>  | <b>ls -al ./profile-launcher</b> command to show the binary executable                |
+    !!! Note
+        The model downloader script is automatically called as part of run-ovms.sh (part of run.sh).
+    
+    ```bash
+    sh ./download_models/getModels.sh --workload ovms
+    ```
+    
+    !!! Warning
+        Depending on your internet connection, this might take some time.
 
-!!! Note
-    Build command may take a while, depending on your internet connection and machine specifications.
 
-!!! Note
-    If the build command succeeds, you will see all the built Docker images files as indicated in the **Check Success** column. If the build fails, check the console output for errors.
+5. Download image file manually (Optional)
```

Comment:
Should we describe why we are downloading this image file?. maybe we can do that in a future refactoring.

Reply:
done